### PR TITLE
Implements the animated property enabling/disabling the animation of …

### LINKED ISSUE #73: Disable animation
Is it possible to stop the rotation animation of the button when it is pressed? I just need the button itself.

### PR DESCRIPTION
**Pull request by araujobarret**
…the component and it's own action items

It is just an idea, probably there is an option to do it in a better way, so please, fell free to review it :smile: 

This closes #73 issue.

### CODE REVIEW

File: component/FloatingActionItem.js [hunk 1]
```diff
@@ -13,12 +13,12 @@ import { getTouchableComponent } from './utils/touchable';
 class FloatingActionItem extends Component {
   constructor(props) {
     super(props);
-
+    console.log('props', props);
```
Comment (santomegonzalo) [inline component/FloatingActionItem.js]: let's remove this console.log :) ... I will be adding some prettier to prevent this error soon

**Comment by santomegonzalo** [comment]
Everything looks perfect... I will test it today to prevent breaking something but I don't think so....

File: component/FloatingActionItem.js (hunk 1 above)
Comment (araujobarret) [reply component/FloatingActionItem.js]: Oh I forgot about it, mb.

Great, thanks!

**Comment by araujobarret** [comment]
I created a commit by github UI to remove the console.log. I am used to gitlab feature to squash the commits on a MR/PR by the UI, do you need that I manually close the PR, do the correction and then create the PR again?

**Comment by araujobarret** [comment]
Still this repo active?

**Comment by santomegonzalo** [comment]
Hi @araujobarret ... yes is still active but I was with a lot of work and moving to a new country... I will take a look and release a new version...

**Pull request merged**